src/ImageSharp/Image{TPixel}.cs

```diff
@@ -18,9 +18,11 @@ namespace SixLabors.ImageSharp
     /// For generic <see cref="Image{TPixel}"/>-s the pixel type is known at compile time.
     /// </summary>
     /// <typeparam name="TPixel">The pixel format.</typeparam>
-    public sealed class Image<TPixel> : Image
+    public class Image<TPixel> : Image
```

Comment:
Without prototyping an actual subclassing use-case, I find this dangerous and unnecessary. We can do the sealed -> non-sealed flip any time, but not the opposite.

Reply:
No worries, will re seal.